schemas/search/2.0/search.json

```diff
@@ -0,0 +1,61 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "$ref": "#/definitions/Search",
+
+  "definitions": {
+    "Search": {
+      "type": "object",
+      "properties": {
+        "languages": { "type": "array", "items": { "$ref": "#/definitions/SearchLanguage" } },
+        "keyboards": { "type": "array", "items": { "$ref": "#/definitions/SearchKeyboard" } },
+        "countries": { "type": "array", "items": { "$ref": "#/definitions/SearchCountry" } },
+        "context": { "$ref": "#/definitions/Context" }
+      }
+    },
+
+    "Context": {
+      "type": "object",
+      "properties": {
+        "range": { "type": "string" },
+        "context": { "type": "string", "enum": ["keyboard", "language", "region"] },
+        "platform": { "type": "string", "enum": ["android", "ios", "linux", "macos", "web", "windows"] },
+        "pageSize": { "type": "number" },
+        "pageNumber": { "type": "number" },
+        "totalRows": { "type": "number" },
+        "totalPages": { "type": "number" }
+      },
+      "required": ["range", "context", "pageSize", "pageNumber", "totalRows", "totalPages"]
+    },
+
+    "SearchCountry": {
+      "type": "object",
+      "properties": {
+        "id": { "type": "string" },
+        "name": { "type": "string" },
+        "area": { "type": "string" },
+        "languages": { "type": "array", "items": { "$ref": "#/definitions/SearchLanguage" } }
+      }
+    },
+
+    "SearchLanguage": {
+      "type": "object",
+      "properties": {
+        "id": { "type": "string" },
+        "name": { "type": "string" },
+        "keyboards": { "type": "array", "items": { "type": "string" } }
+      }
+    },
+
+    "SearchKeyboard": {
+      "anyOf": [
+        { "$ref": "/keyboard_info.source/1.0.6/keyboard_info.source.json#/definitions/KeyboardInfo" },
+        {
+          "type": "object",
+          "properties": {
+            "match": { "type": "object" }
```

Comment:
Am I missing something, or is the schema for `"match"` rather incomplete?

Reply:
Noted. Yes. It is fairly light. I will fix that in a future PR.